src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const getPartner = sex === 'f'
```

Comment:
get sounds like an action, good prefix for function, but not variable

I'd like you to name it as partnerInfo or something like this, you decide